shopfloor_mobile_base/static/wms/src/css/main.css

```diff
@@ -457,6 +457,10 @@ main.v-content > .v-content__wrap > .header .container {
     line-height: 1.1rem;
 }
 
+.v-card .v-card__title-smaller {
+    font-size: 0.9rem;
```

Comment:
this does not ensure the name fits... no?

Reply:
No but it is more likely. Maybe the name of the pr should be changed ?

Reply:
@TDu Can you try with something similar to this exemple: https://jsfiddle.net/hymbof7o/

Reply:
I did not know about the `vw` unit. It stands for View Width (1vw is 1% of the width of the viewport). That will not help.
As for the `whitespace: nowrap;`  it will not wrap the text to the next line, but it will not dynamically resize the content to fit.

Reply:
Have we considered @container?
https://developer.mozilla.org/en-US/docs/Web/CSS/@container